test/acceptance/test/testsuite_common_test.go

```diff
@@ -48,7 +47,7 @@ func TestAcceptance(t *testing.T) {
 	}
 
 	RegisterFailHandler(Fail)
-	gomega.RegisterFailHandler(GomegaFail)
+	//gomega.RegisterFailHandler(GomegaFail)
```

Comment:
```suggestion
```
Nit; go ahead and delete the line

Reply:
I'll address this in another PR, I forgot changing this temporary code. This is the main reason why I created that PR. Here it is https://github.com/weaveworks/weave-gitops/pull/1252